infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,54 @@
+import * as ec2 from "@aws-cdk/aws-ec2";
+import * as ecs_patterns from "@aws-cdk/aws-ecs-patterns";
+import * as cdk from "@aws-cdk/core";
+import * as ecs from "@aws-cdk/aws-ecs";
+
+export class InfrastructureStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // TODO we need to figure out best practices for standing up the DB
+    // either in the same CF stack, or in a different one.
+    // what's more, we're going to need to figure out how to create users
+    // in it, and then how to get it initialized to have all of the tables
+    // created for us.
```

Comment:
Any reason not to let our application create the tables for us? I'd image we would want to manage DB migrations near our code?